lib/base-compiler.ts

```diff
@@ -1049,11 +1049,13 @@ export class BaseCompiler {
         const execOptions = this.getDefaultExecOptions();
         execOptions.maxOutput = 1024 * 1024 * 1024;
 
+        const compile_start = performance.now();
```

Comment:
Can you use the numbers that exec() returns here? To make it more consistent with other timings and not include too much js overhead